### PR TITLE
Discontinue Kubeflow Addon

### PR DESCRIPTION
**Pull request by DomFleischmann**
The Charmed Kubeflow team has decided to discontinue the Microk8s Kubeflow
Addon. This is due to the upstream community struggling to keep up to
date with the latest Kubernetes versions. Additionally the Charmed
Kubeflow team wants to standardize on one unique installation experience
instead of having to support multiple ones.

Kubeflow will still be available on Microk8s by following these new
installation instructions:
https://charmed-kubeflow.io/docs/quickstart

### CODE REVIEW

File: microk8s-resources/actions/enable.kubeflow.sh [hunk 1]
```diff
@@ -146,6 +146,18 @@ def wait_ready(debug: bool, extra: bool = True):
         )
 
 
+def report_discontinued():
+    """The Kubeflow addon is officially deprecated.
+    Fail gracefully informing the user where to install kubeflow now.
+    """
+
+    print("")
+    print("This addon is no longer supported, instead please follow the quick start guide to install Charmed Kubeflow on Microk8s: ")
```
Comment (grobbie) [inline microk8s-resources/actions/enable.kubeflow.sh:155]:
```suggestion
    print("This addon is no longer supported, instead please follow the quick start guide to install Charmed Kubeflow on MicroK8s: ")
```

**Review by grobbie — CHANGES_REQUESTED** [review]

**Comment by ktsakalozos** [comment]
Hi @DomFleischmann, do you plan to address the failing Kubeflow tests?

**Comment by DomFleischmann** [comment]
> Hi @DomFleischmann, do you plan to address the failing Kubeflow tests?

I think we might as well disable them, what do you think? Should I just remove them?

**Comment by ktsakalozos** [comment]
> Should I just remove them?

Yes, that would be best.

**Comment by ktsakalozos** [comment]
Thank you for this PR @DomFleischmann

**Pull request merged**